client/blocks/reader-recommended-sites/index.jsx

```diff
@@ -49,7 +49,7 @@ export class RecommendedSites extends React.PureComponent {
 						map(
 							sites,
 							( site, index ) => {
-								const siteId = site.siteId;
+								const siteId = site.siteId || site.blogId;
```

Comment:
That'd be nice. It'd be good to deal with one consistently throughout Reader, and we generally go with blogId.

They usually mean the same thing, except when they don't...